frontend/src/components/mine/Variances/MineVarianceTable.js

```diff
@@ -50,6 +61,8 @@ export class MineVarianceTable extends Component {
     variances.map((variance) => ({
       key: variance.variance_guid,
       variance,
+      mineName: variance.mine_name || null,
+      mineGuid: variance.mine_guid || null,
```

Comment:
Is this to convert empty strings and undefined to null?

Reply:
`MineVarianceTable` is being used to display the results from `/variances` and `/mines/guid/variances`. The former includes the `mine_name` and `mine_guid` int he payload. 

I was trying to avoid the error "cannot read mine_name of undefined"